hail/python/hail/experimental/vcf_combiner/vds_combiner.py

```diff
@@ -0,0 +1,190 @@
+from typing import List
```

Comment:
I think this should be in the `hl.vds` module! We should phase out the experimental module when the new stuff is documented,tested, and performant.